_posts/2024-02-23-gsoc2024.md

```diff
@@ -0,0 +1,171 @@
+| 1       | [Generalise Groups]({{ site.github.wiki }}/GSoC-2024-Project-Ideas#project-1-bead-and-ring-groups)                                         | hard     | 350 hours   | Generalise concept of groups                                                                   | Python, NetworkX, Molecular modeling                  | @richardjgowers, @yuxuanzhuang |
+| 2       | [Extend MDAnalysis Interoperability]({{ site.github.wiki }}/GSoC-2024-Project-Ideas#project-2-extend-interoperability)                  | medium     | 350 hours   | Extend converters module to other relevant packages                                            | Python, Molecular modeling                      | @hmacdope, @yuxuanzhuang, @xhgchen |
+| 3       | [On the fly transformations]({{ site.github.wiki }}/GSoC-2024-Project-Ideas#project-3-on-the-fly-transformations)                  | medium/hard     | 175/350 hours   |  Expand on code for on-the-fly transformations through documentation, product research or performance optimization | Python, Molecular modeling, Computational chemistry, Scientific writing, Performance profiling                      | @richardjgowers, @cbouy, @hmacdope, @xhgchen |
+| 4       | [2D visualization for small molecules]({{ site.github.wiki }}/GSoC-2024-Project-Ideas#project-4-2d-visualization-for-small-molecules)                                         | easy     | 90 hours  | Add basic 2D visualization functionalities for small molecule groups in notebooks                                                                   | Python, basic knowledge of MDAnalysis and RDKit                 | @cbouy, @hmacdope, @xhgchen |
+| 5       | [Better interfacing of Blender and MDAnalysis]({{ site.github.wiki }}/GSoC-2024-Project-Ideas#project-5-better-interfacing-of-blender-and-mdanalysis)                                         | medium     | 350 hours   | Add functionality to visualize simple MDAnalysis results in Blender                                                                  | Python, basic knowledge of MDAnalysis, familiarity with Blender ideal                  | @BradyAJohnston, @yuxuanzhuang |
+
+
+## Information for prospective GSoC Contributors
+
+You must meet our [own requirements] if you want to be a GSoC
+Contributor with MDAnalysis this year (read all the docs behind these
+links!) and read the points below for the application process. You must also meet the [eligibility criteria]. Our [GSoC
+FAQ][] collects common questions from applicants.
+
+In short:
+Agree to be bound by our [Code of Conduct];
+Familiarize yourself with MDAnalysis;
+Submit a short pre-proposal — 
+**If you have been invited to proceed with your application (based on your pre-prosal)** have a pull request merged in MDAnalysis.
```

Comment:
Make bullet points?

Reply:
Added numbered lists - the formatting must not have worked correctly when I initially copied this over from the google doc.